config/application.rb

```diff
@@ -4,25 +4,20 @@
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
+# require "active_storage/engine"
```

Comment:
do we need these require comments?

Reply:
Personal opinion: these comments makes it easy to know that the active storage, action cable etc are disabled. Makes it also easy to enable them.